requirements.txt

```diff
@@ -1,7 +1,9 @@
 asynctest
 colorama
 coverage
-requests_oauthlib
+tox
```

Comment:
Why are we re-adding `tox` here?

Reply:
I noticed that you had dev dependencies in requirements.txt as well, so i added it in while i was testing this branch. Did you not intend to have dev dependencies in that file?